sys/targets/targets.go

```diff
@@ -184,7 +184,10 @@ var List = map[string]map[string]*Target{
 			PtrSize:     8,
 			PageSize:    4 << 10,
 			CFlags:      []string{"-m64"},
-			CrossCFlags: []string{"-m64", "-static"},
+			CrossCFlags: []string{"-m64", "-static",
+				"--sysroot", os.ExpandEnv("${SOURCEDIR}../dest/"),
```

Comment:
People (me) will set SOURCEDIR without the trailing `/` and the report that build is not working. In other places in this file we use "${SOURCEDIR}/dir" and double `/` is ignored on linux, so it will always work. Let's do the same here.
A completely proper way to do this would include `filepath.Join` or `filepath.Clean` functions, but that may be overkill for now.

Reply:
Fixed.